index.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="title">
+    <h1 class="title-text">Front-End Tools</h1>
+  </div> 
+  
+  <div class="techs-container">
+    <div class="tech">
+      <img class="tech-img" src="img/react_icon.svg">
+      <h4 class="tech-text tech-text-title">ReactJS</h4>
+      <p class="tech-text tech-text-release">Release: 2013</p>
```

Comment:
No need to add so much classes to the element. Just add the one to target the element, in this case, simply use the class `.tech-release` or `.release`, either way is OK